### DISCUSSION TITLE
JSONB operators @@ and @?

### DISCUSSION THREAD

**Discussion by MarkBrook** [sqlalchemy/sqlalchemy#7146] [Usage Questions] [answered]
Is there any way to use `@@` and `@?` using `JSONB` comparator? Or maybe any other approach except comparator?
Thanks!

**Comment by CaselIT** [comment, marked as answer]
Hi,

I think you should be able to use [op](https://docs.sqlalchemy.org/en/14/core/sqlelement.html#sqlalchemy.sql.expression.Operators.op) as in `table.c.jsonb_col.op('@@')('something')`

**Reply by CaselIT** [reply]
Maybe they could be a candidate for addition in jsonb. They seem to be a newer addition to jsonb, seems from v12, so that's probably why they are missing from the type

Not sure what name they have. `atat` and `atquestion` does not seem appropriate

**Reply by MarkBrook** [reply]
Thank you for help!

**Reply by CaselIT** [reply]
@zzzeek mike do you think we should add these?

**Reply by zzzeek** [reply]
if pg has added lots of new operators we would do a sweep, sure

**Reply by CaselIT** [reply]
I've added https://github.com/sqlalchemy/sqlalchemy/issues/7147
